lib/poker_hands/deck.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'card'
+
+class Deck
+  def initialize
+    @hand_size = 5
```

Comment:
Is there a reason why a `Deck` should know about the size of a hand?

Reply:
Would you advocate for the `deal` method to have a parameter to receive hand_size?

Reply:
I would consider moving the idea of dealing cards out of this class entirely.